src/backend/models/pageOrder.ts

```diff
@@ -75,6 +75,28 @@ class PageOrder {
     return Promise.all(docs.map(doc => new PageOrder(doc)));
   }
 
+  /**
+   * Returns only root page's order
+   *
+   * @returns {Promise<PageOrder[]>}
+   */
+  public static async getRootPageOrder(): Promise<PageOrder> {
+    const docs = await this.getAll();
+
+    return docs.filter(doc => doc.page === '0')[0];
+  }
+
+  /**
+   * Returns only child page's order
+   *
+   * @returns {Promise<PageOrder[]>}
+   */
+  public static async getChildPageOrder(): Promise<PageOrder[]> {
+    const docs = await this.getAll();
+
+    return docs.filter(doc => doc.page !== '0');
```

Comment:
here we can also filter documents while querying database

Reply:
Fixed